cli.ts

```diff
@@ -309,6 +310,61 @@ const postInitCLI = (cliConfig: CLIConfig, env: EnvVars, args: DenoArgs, parsedA
 				default: false,
 				boolean: true,
 			})
+			.command(
+				'add-contract <sourceFile>',
```

Comment:
Hyphenation here breaks from our current standard with space-separated words (`taq list accounts`, `taq start sandbox`, etc)

Reply:
I remember having discussions about turning all of these space-separated words into hyphenated ones because `taq start sand` (should fail but don't) works the same as `taq start sandbox` currently.

Reply:
That is actually a good failure, we want the cli to be as resilient to user error as possible